public/js/monitor.js

```diff
@@ -81,59 +81,70 @@ function addInvalidMessage() {
   addClass(".input-group", "invalid");
 }
 
-function enableBtnIfEmailValid(e) {
-  removeInvalidMessage();
-  const emailBtn = document.getElementById("submit-email");
-  if (isValidEmail(e.target.value)) {
-      emailBtn.disabled = false;
-      removeInvalidMessage();
-  } else {
-      emailBtn.disabled = true;
-      if(e.code === "Enter") {
-        addInvalidMessage();
-      }
-  }
-}
-
-function enableBtnIfInputEmpty(e) {
-  const emailBtn = document.getElementById("submit-email");
-  if (!document.querySelector("input[name=email]").value) {
-      emailBtn.disabled = false;
-      removeInvalidMessage();
-  } else {
-      enableBtnIfEmailValid(e);
-  }
+const removeModalThings = function() {
```

Comment:
Is there a benefit to this pseudo-anon function syntax versus vanilla `function removeModalThings() { .. }`?